.resources/strings.json

```diff
@@ -243,104 +272,116 @@
 		"pl": "Okno",
 		"ja": "ウィンドウ",
 		"es": "Ventana",
-		"de": "Fenster"
+		"de": "Fenster",
+		"pt-BR": "Janela"
 	},
 	"menu.window.minimize-mac": {
 		"en": "Minimize",
 		"fr": "Placer dan le Dock",
 		"pl": "Minimalizuj",
 		"ja": "最小化",
 		"es": "Minimizar",
-		"de": "Im Dock ablegen"
+		"de": "Im Dock ablegen",
+		"pt-BR": "Minimizar"
 	},
 	"menu.window.minimize": {
 		"en": "Minimize",
 		"fr": "Réduire",
 		"pl": "Minimalizuj",
 		"ja": "最小化",
 		"es": "Minimizar",
-		"de": "Minimieren"
+		"de": "Minimieren",
+		"pt-BR": "Minimizar"
 	},
 	"menu.window.zoom-mac": {
 		"en": "Zoom",
 		"fr": "Réduire/agrandir",
 		"pl": "Zoom",
 		"ja": "ズーム",
 		"es": "Maximizar",
-		"de": "Zoomen"
+		"de": "Zoomen",
+		"pt-BR": "Ampliar"
 	},
 	"menu.window.zoom": {
 		"en": "Zoom",
 		"fr": "Agrandir",
 		"pl": "Zoom",
 		"ja": "ズーム",
 		"es": "Maximizar",
-		"de": "Zoomen"
+		"de": "Zoomen",
+		"pt-BR": "Ampliar"
 	},
 	"menu.window.front": {
-		"en": "Brint All to Front",
+		"en": "Bring All to Front",
 		"fr": "Tout ramener au premier plan",
 		"pl": "Umieść wszystko na wierzchu",
 		"ja": "全てを前面に表示",
 		"es": "Traer al frente",
-		"de": "Alle nach vorn bringen"
+		"de": "Alle nach vorn bringen",
+		"pt-BR": "Trazer tudo para frente"
 	},
 	"menu.window.window": {
 		"en": "Window",
 		"fr": "Fenêtre",
 		"pl": "Okno",
 		"ja": "ウィンドウ",
 		"es": "Ventana",
-		"de": "Fenster"
+		"de": "Fenster",
+		"pt-BR": "Janela"
 	},
 	"menu.window.close": {
 		"en": "Close",
 		"fr": "Fermer",
 		"pl": "Zamknij",
 		"ja": "閉じる",
 		"es": "Cerrar",
-		"de": "Schließen"
-	},
+		"en": "Schließen",
+		"pt-BR": "Fechar"
 
 	"menu.help.name": {
```

Comment:
It's highlighting this colon as red too, maybe I pasted in a wrong character before? Can you change it to a normal colon character

Reply:
All red highlights fields were a bug from my text editor lint, [micro](https://github.com/zyedidia/micro).